asciidoctorj-core/src/main/java/org/asciidoctor/ast/InlineImpl.java

```diff
@@ -1,38 +1,36 @@
 package org.asciidoctor.ast;
 
 import org.jruby.Ruby;
+import org.jruby.runtime.builtin.IRubyObject;
 
 public class InlineImpl extends AbstractNodeImpl implements Inline  {
 
-    protected Inline delegate;
-
-    public InlineImpl(Inline delegate, Ruby ruby) {
-        super(delegate, ruby);
-        this.delegate = delegate;
+    public InlineImpl(IRubyObject delegate) {
+        super(delegate);
     }
 
     @Override
     public String render() {
-        return delegate.render();
+        return getString("render");
     }
 
     @Override
     public String convert() {
-        return delegate.convert();
+        return getString("convert");
```

Comment:
render() and convert( ) should return an Object (unless we are waiting on #234).